src/controller/summayGeneration.ts

```diff
@@ -0,0 +1,40 @@
+import { Request, Response } from "express";
+import * as openai from "openai";
+import { ChatCompletionRequestMessageRoleEnum } from "openai";
+
+const configuration = new openai.Configuration({
+  apiKey: process.env.OPENAI_API_KEY,
+});
+
+const openaiApi = new openai.OpenAIApi(configuration);
+
+const messages: {
+  role: ChatCompletionRequestMessageRoleEnum;
+  content: string;
+}[] = [
+  {
+    role: ChatCompletionRequestMessageRoleEnum.System,
+    content: "write a summary",
+  },
+];
+
+export const summaryGeneration = async (req: Request, res: Response) => {
+  const ocrContent = req.body.ocrContent;
+  messages.push({
+    role: ChatCompletionRequestMessageRoleEnum.User,
+    content: ocrContent,
+  });
+  try {
+    const completion = await openaiApi.createChatCompletion({
+      model: "gpt-3.5-turbo-0301",
```

Comment:
move max_token and model into ENV, so we can update it according to our needs,
and please add this .env.example as well...